src/status_im/chat/models/loading.cljs

```diff
@@ -69,15 +68,15 @@
 
 (rf/defn handle-mark-all-read-successful
   {:events [::mark-all-read-successful]}
-  [cofx]
-  (activity-center/notifications-fetch-unread-count cofx))
+  [_]
+  {:dispatch [:activity-center.notifications/fetch-unread-count]})
```

Comment:
FYI: I had to switch to `:dispatch` because the compiler failed due to a circular dependency.

Reply:
why not dispatch `fetch-unread-count` directly?